geometry_msgs/Pose2D goal_position
---
geometry_msgs/Pose2D[] path